hack/ci-deploy.sh

```diff
@@ -15,10 +15,10 @@ if [[ -z ${1+x} ]]; then
     exit 1
 fi
 
-ARTIFACTS_DIR=${ARTIFACTS_DIR:-$( mktemp -d )}
```

Comment:
I assume this is coming from https://github.com/scylladb/scylla-operator/pull/1922, please rebase

Reply:
weird, fixed with a clean rebase